soh/src/code/z_play.c

```diff
@@ -2318,7 +2318,10 @@ void Play_PerformSave(PlayState* play) {
         } else {
             Save_SaveFile();
         }
-        if (CVarGetInteger("gAutosave", AUTOSAVE_OFF) != AUTOSAVE_OFF) {
+        uint8_t isTriforceHuntSave =
+            gSaveContext.n64ddFlag && gSaveContext.triforcePiecesCollected == Randomizer_GetSettingValue(RSK_TRIFORCE_HUNT_PIECES_REQUIRED) &&
+            Randomizer_GetSettingValue(RSK_TRIFORCE_HUNT);
```

Comment:
`isTriforceHuntSave` seems like an incorrect name here, ans it's really a bool that's true if all of the required pieces are collected, not always when in a triforce hunt save.

also, this seems to only be used for displaying the "game saved" text?

Reply:
You're right, going to rename to `triforceHuntCompleted`.

And yes, it's just for showing the text. The save happens anyway but it usually only shows the text when autosave is also on, but I want to show the text for this specific save regardless of what auto save is set to.

Reply:
one possible unintended side effect: every manual save after getting all the triforce pieces will show the text. not a dealbreaker, just an oddity.